CHANGES.rst

```diff
@@ -25,6 +25,8 @@ pipeline
 
 - Refactor the ``Image3Pipeline`` to use ``stpipe`` infrastructure. [#4990]
 
+- Blend headers just from each input science model, not every integration. [#5007]
```

Comment:
Should mention that this change was applied in the `Coron3Pipeline` (or `calwebb_coron3`)